tests/unit/controllers/problemSet.test.ts

```diff
@@ -162,7 +162,7 @@ describe("Problem sets controller tests", () => {
             );
         });
 
-        it("status 404: returns an appropriate response if user doesn't exist", async () => {
+        it("status 404: returns an appropriate response if problem set doesn't exist", async () => {
             stubs.problemSetDB.find.resolves(null);
```

Comment:
There is no need to resolve a stub to return null. When you stub a method, by default, it returns null.